my-app/src/file.js

```diff
@@ -0,0 +1,82 @@
+import React from "react"
+import {SketchField, Tools} from 'react-sketch';
+import { SketchPicker } from 'react-color';
+import Modal from "react-responsive-modal";
+
+import './css/file.css'
+
+import Chat from './chat'
+import voicePic from './voice chat icon.png'
+import 'react-sliding-pane/dist/react-sliding-pane.css';
+
+class File extends React.Component{
+	state = {
+		open: false,
+		penColor: 'black',
+		tool: Tools.Pencil,
+		toolBox: ["Pencil", "Select", "Circle", "Rectangle", "Pan"]
+	};
+
+	handleChangeComplete = (color) => {
+		this.setState({ penColor: color.hex });
+	};
+
+	handleChangeTools = (item) => {
+		this.setState({tool: item});
+		console.log("ChosenTool: ", this.state.tool)
+	};
+
+	onOpenModal = () => {
+		this.setState({ open: true });
+	};
+
+	onCloseModal = () => {
+		this.setState({ open: false });
+	};
+
+
+	render(){
+		const { open } = this.state;
+		const {tool} = this.state;
+		return(
+			<div>
+				<h1 id = 'filename'>Filename</h1>
+				<a href="/home" className="myButton">Home</a>
+				<div id = 'filenameDiv'/>
+				<div id = "spaceDiv"/>
+				<div class = "row">
+					<button class = "row" id="buttonForCanvas" onClick={this.onOpenModal}>
+						<div id = "colorBox" style = {{backgroundColor: this.state.penColor}}></div>
+						<div id = "colorText">Color</div>
+					</button>
+
+					<button id = 'dropDownButton' onClick={()=>this.setState({tool: Tools.Pencil})}> Pencil</button>
```

Comment:
Oh I see you're using your ids like classes. Every ID should only occur once. You should not have more than one element with id `dropDownButton`. The IDs here should be classes:

`class='dropDownButton'` not `id='dropDownButton'`